content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<ul><li>I think you meant matrix multiplication <strong>function</strong> <code>np.dot</code>  (instead of *module*)</li><li>In the activation function description I think you meant "inputs and outputs"  instead of "inputs and inputs" :)</li><li>"In this example, you will use a method called dropout (<a href="https://en.wikipedia.org/wiki/Dilution_(neural_networks))" target="_blank">dilution</a> that " (I think there is a formatting error here with the parenthesis?)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
- Good catch :) Also changed "`np.random.random()` function" and "`np.random.randint()` function".
- Nice one. Changed to "inputs and outputs"
- There's a weird rendering issue in some notebook viewers - maybe because the URL ends with a bracket `[...](https://en.wikipedia.org/wiki/Dilution_(neural_networks))`. Changed it to:

```diff
- ... dropout ([dilution](https://en.wikipedia.org/wiki/Dilution_(neural_networks))) ...
+ dropout — [dilution](https://en.wikipedia.org/wiki/Dilution_(neural_networks)) — ...
```

Reply:
<p>The regularization link seems to be missing a parenthesis at the end. Thanks for the other fixes :)</p>

Reply:
😄 I think I found the missing bracket.